charts/scalardb-analytics-postgresql/templates/scalardb-analytics-postgresql/deployment.yaml

```diff
@@ -0,0 +1,154 @@
+          volumeMounts:
+            - mountPath: /etc/database.properties
+              name: database-properties-volume
+              subPath: database.properties
+          resources:
+            {{- toYaml .Values.scalardbAnalyticsPostgreSQL.resources | nindent 12 }}
+          startupProbe:
+            exec:
+              command:
+                - psql
+                - -d
+                - "{{ .Values.scalardbAnalyticsPostgreSQL.postgresql.databaseName }}"
+                - -U
+                - postgres
+                - -c
+                - "SELECT 1"
+            failureThreshold: 60
+            periodSeconds: 5
+          livenessProbe:
+            exec:
+              command:
+                - psql
+                - -d
+                - "{{ .Values.scalardbAnalyticsPostgreSQL.postgresql.databaseName }}"
+                - -U
+                - postgres
+                - -c
+                - "SELECT 1"
+            failureThreshold: 3
+            periodSeconds: 10
+            successThreshold: 1
+            timeoutSeconds: 1
+        - name: schema-importer
+          securityContext:
+            {{- toYaml .Values.schemaImporter.securityContext | nindent 12 }}
+          image: "{{ .Values.schemaImporter.image.repository }}:{{ .Values.scalardbAnalyticsPostgreSQL.image.tag | default .Chart.AppVersion }}"
+          command: [/entrypoint.sh]
+          args:
+            - java
+            - -jar
+            - /app.jar
+            - import
+            - --host
+            - localhost
+            - --port
+            - "5432"
+            - --database
+            - "{{ .Values.scalardbAnalyticsPostgreSQL.postgresql.databaseName }}"
+            - --user
+            - postgres
+            - --password
+            - "{{ .Values.scalardbAnalyticsPostgreSQL.postgresql.password }}"
+            - --config
+            - /etc/database.properties
+            {{- range .Values.schemaImporter.namespaces }}
+            - --namespace
+            - {{ . | quote }}
+            {{- end }}
+          env:
+            - name: SCHEMA_IMPORTER_MAX_RETRY_COUNT
+              value: "{{ .Values.schemaImporter.entrypointShell.maxRetryCount }}"
+            - name: SCHEMA_IMPORTER_RETRY_INTERVAL
+              value: "{{ .Values.schemaImporter.entrypointShell.retryInterval }}"
+          volumeMounts:
+            - mountPath: /etc/database.properties
+              name: database-properties-volume
+              subPath: database.properties
+            - mountPath: /entrypoint.sh
+              name: entrypoint-shell-volume
+              subPath: entrypoint.sh
+          {{- with .Values.scalardbAnalyticsPostgreSQL.extraVolumeMounts }}
+            {{- toYaml . | nindent 12 }}
+          {{- end }}
+      volumes:
+        - configMap:
+            defaultMode: 0440
+            name: {{ include "scalardb-analytics-postgresql.fullname" . }}-database-properties
+          name: database-properties-volume
```

Comment:
The `ScalarDB Analytics with PostgreSQL` container and `Schema Importer` container share the same `database.properties` file.